src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -59,17 +59,19 @@
                                lang="en">
                         <label class="custom-file-label" for="import-csv">{{ importedFilename }}</label>
                     </div>
-                    <div v-if="validationErrors.length"
-                         id="import-validation-errors"
-                         class="text-danger small error-message mt-2 ">
-                        <div class="col-sm-2 d-inline-block"></div>
-                        <div class="col-sm-6 d-inline-block pl-0">
-                            Unable to import from csv:
-                            <div v-for="error in validationErrors" class="import-validation-error ml-2">
+                    <b-alert :show="!!validationErrors.length"
+                             id="import-validation-errors"
+                             dismissible
+                             variant="danger"
+                             class="col-sm-10 mt-4"
+                             @dismissed="validationErrors=[]">
+                        Failed to import from csv. The following issues were found:
+                        <ul class="py-0 my-0 ml-2" :style="{listStyleType: 'disc'}">
+                            <li v-for="error in validationErrors" class="import-validation-error">
                                 {{error.message}}
-                            </div>
-                        </div>
-                    </div>
+                            </li>
+                        </ul>
```

Comment:
We should probably wrap this snippet in a div or span then add pr-2 to its class for some starting space.

Reply:
Not sure what you mean by 'starting space'. `pr-2` is right-padding so that wouldn't really be 'starting space' - do you want more padding on both left and right? Would that be instead of or as well as reducing the width to match the inputs? Won't be much room for extra padding if we do that I think..